File: app/src/main/java/com/song/deviceinfo/ui/emulator/EmulatorViewModel.java
package com.song.deviceinfo.ui.emulator;

import android.content.Context;

import com.song.deviceinfo.info.EmulatorInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/7/9.
 */
public class EmulatorViewModel extends NormalViewModel {

    public List<Pair<String, String>> getEmulatorInfo(Context context) {
        return EmulatorInfo.getEmulatorInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/emulator/EmulatorViewModel.kt
package com.song.deviceinfo.ui.emulator

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.EmulatorInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/7/9.
 */
class EmulatorViewModel : NormalViewModel() {
    fun getEmulatorInfo(context: Context?): List<Pair<String, String>> {
        return EmulatorInfo.getEmulatorInfo(context!!)
    }
}
